ISSUE_TEMPLATE.md

```diff
@@ -1 +1,47 @@
+<!--
+Please use the below format to report the issue in a better way.
+To check any option, replace the space between the "[" and "]" with a "x".
+-->
 
+#### Nature of Issue ?
+
+- [ ] Found a Bug
+- [ ] Existing Feature Enhancement
+- [ ] New Feature Request
+
+#### Most Appropriate Sub Area ?
+
+- [ ] Color
+- [ ] Core
+- [ ] Events
+- [ ] Image
+- [ ] IO
+- [ ] Math
+- [ ] Typography
+- [ ] Utilities
+- [ ] WebGL
+
+<!--
+	Fill the following sections as suitable, and delete the part which is not applicable.
+-->
+
+<!-- If you found a bug, then please consider the following points -->
```

Comment:
can we say "if you know them" in there somehow? I don't want to make beginners that don't know what an OS or how to find the version number feel dumb or like they can't post an issue